easybuild/easyconfigs/m/MMseqs2/MMseqs2-13-45111-gompi-2020b.eb

```diff
@@ -17,10 +17,6 @@ builddependencies = [('CMake', '3.18.4')]
 
 dependencies = [('bzip2', '1.0.8')]
 
-separate_build_dir = True
-
-configopts = '-DCMAKE_BUILD_TYPE=RELEASE'
```

Comment:
@Flamefire `build_type = 'RELEASE'` is missing here?

Reply:
Or maybe it's not needed, since `CMakeMake` uses `-DCMAKE_BUILD_TYPE=Release` by default?

Reply:
Ah, you mentioned it in the description... Good to go then :+1: